frontend/src/pages/articles/components/article-card/article-card.tsx

```diff
@@ -50,7 +54,10 @@ const ArticleCard: React.FC<Properties> = ({
       <div className={styles.body}>
         <div className={styles.articleInfo}>
           <h4 className={styles.title}>{title}</h4>
-          <p className={styles.text}>{text}</p>
+          <p
+            className={styles.text}
+            dangerouslySetInnerHTML={getDangerousHtmlObject(text)}
+          ></p>
```

Comment:
Not sure if helper to get prop for `dangerouslySetInnerHTML` is necessary
```suggestion
          <article
            className={styles.text}
            dangerouslySetInnerHTML={{ __html: sanitizeHtml(htmlContent) }}
          />
```